src/topology/metric_space/pi_Lp.lean

```diff
@@ -229,6 +229,11 @@ lemma norm_eq {p : ℝ} {hp : 1 ≤ p} {α : ι → Type*}
   [∀i, normed_group (α i)] (f : pi_Lp p hp α) :
   ∥f∥ = (∑ (i : ι), ∥f i∥ ^ p) ^ (1/p) := rfl
 
+lemma norm_eq_of_nat {p : ℝ} {hp : 1 ≤ p} {α : ι → Type*}
```

Comment:
I added this lemma because of the notational confusion between `^` when encoding `rpow` or `monoid.pow`. For working with `nat` powers, this will allow for cleaner rewrites, with easy discharging with `norm_num`. I couldn't figure out how to use `. tactic.norm_num` as the out_param there.